petclinic-frontend/src/features/inventories/InventoriesListTable.tsx

```diff
@@ -547,15 +549,50 @@ export default function InventoriesListTable(): JSX.Element {
           </div>
         </>
       )}
-      <button className="btn btn-primary" onClick={toggleAddSupplyModal}>
-        Add Supply
+      <button className="btn btn-primary" onClick={toggleAddProductModal}>
+        Add Product
       </button>
-      {showAddSupplyModal && (
-        <AddSupplyForm
-          onClose={toggleAddSupplyModal}
-          onSubmit={handleAddSupplySubmit}
+      {showAddProductModal && (
+        <AddProductForm
+          onClose={toggleAddProductModal}
+          onSubmit={handleAddProductSubmit}
         />
       )}
+      <div className="card-container-custom">
+        {inventoryList.map(inventory => (
+          <div
+            className="card"
+            key={inventory.inventoryName}
+            onClick={() =>
+              navigate(`/inventory/${inventory.inventoryId}/products`)
+            }
+            style={{ cursor: 'pointer' }}
+          >
+            <div className="image-container">
+              <img
+                src={inventory.inventoryImage}
+                alt={inventory.inventoryName}
+                className="card-image"
+                onError={(e: React.SyntheticEvent<HTMLImageElement, Event>) => {
```

Comment:
I really like the way you implemented a backup image in case the inventoryImage is not fetched correctly. I think it's really well coded and it's a good way to prevent error messages form displaying on the page.